packages/react-form/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [0.9.0] - 2020-12-03
 
+- Added new functionality to `useDynamicList`. Added the ability to dynamically add more than one list item and the ability to pass in an argument into the dynamic list factory.
```

Comment:
This is listed as already released in 0.9.0, but I think you meant to place it above. I'll move the changelog ☝️ when releasing 0.10.0.

Reply:
thanks! @gmcgibbon